### ISSUE TITLE
What are the best parameters?

### ISSUE THREAD

**Issue by stephane-archer** [clcarwin/focal_loss_pytorch#3]
gamma, alpha and size_average.
What are your recommendation when you init the loss function?
How gamma and alpha affect the result?

**Comment by bonlime** [comment]
You can find it in original paper: https://arxiv.org/pdf/1708.02002.pdf
Authors used gamma=2, alpha=0.25

**Comment by PaParaZz1** [comment]
As usual, gamma=2, and you should adjust alpha to match you training data distribution approximately.
